fastlane/docs/Codesigning/Troubleshooting.md

```diff
@@ -0,0 +1,67 @@
+# Debugging codesigning issues
+
+This guide will help you resolve the most common code signing errors.
+
+## Error message
+
+Make sure to find the exact error message in your build output. Many times the error message actually tells you how to resolve the issue. 
+
+To get even more details, try archiving using Xcode 8. 
+
+## Different machine
+
+Does code signing work on a different Mac, but not on yours? If so, chances are high you miss the latest private key, the certificate or the provisioning profile. Also follow the [Keychain](#keychain) part of this document to make sure you don't have any expired certificates installed.
```

Comment:
"If so, chances are high you miss the latest private key, the certificate or the provisioning profile" miss should be "are missing" or "lack" or "don't have"